spec/WFST.js

```diff
@@ -1,14 +1,14 @@
 /**
  * Created by PRadostev on 18.02.2015.
  */
-describe('WFST', function () {
```

Comment:
please restore this spaces between `function` and `(`. They presented in all project.